hw/sensor/src/sensor_shell.c

```diff
@@ -591,7 +572,7 @@ sensor_cmd_read(char *name, sensor_type_t type, struct sensor_poll_data *spd)
 err:
     os_sem_release(&g_sensor_shell_sem);
```

Comment:
I don't think this call to `os_sem_release()` is necessary (I know this was here before your change).